### PR TITLE
Problem: `x/group` module not added and tested

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Added `x/group` with integration tests

### CODE REVIEW

File: integration_tests/test_group.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            home=cli.data_dir,
+            node=cli.node_rpc,
+            keyring_backend="test",
+            chain_id=cli.chain_id,
+        )
+    )
+
+    assert rsp["code"] == 0, rsp["raw_log"]
+
+    # check if the proposal executed successfully
```
Comment (tomtau) [inline integration_tests/test_group.py:204]: two more assertion to check that the amount was transferred successfully (deducted from the group / sender, and added to the receiver)?

**Review by tomtau — APPROVED** [review]
is there some reference app for group wiring? simapp?

**Comment by devashishdxt** [comment]
> is there some reference app for group wiring? simapp?

Yes. `simapp`.

File: integration_tests/test_group.py (hunk 1 above)
Comment (devashishdxt) [reply integration_tests/test_group.py:207]: Done

**Pull request merged**